meson.build

```diff
@@ -1,10 +1,10 @@
 project('serpapi-search-cpp', 'cpp',
   version : '0.1',
-  default_options : ['warning_level=3', 'cpp_std=c++14'])
+  default_options : ['warning_level=3', 'cpp_std=c++17'])
 
 dep = [
-    dependency('curl'),
-    dependency('rapidjson')
+    dependency('libcurl'),
+    dependency('RapidJSON')
```

Comment:
On Fedora and Ubuntu, there are no package configs for `curl` and `rapidjson`. They are named `libcurl` and `RapidJSON`.

```bash
$ pkg-config --list-all | grep -iE 'rapidjson|curl'
libcurl                        libcurl - Library to transfer files with ftp, http, etc.
RapidJSON                      RapidJSON - A fast JSON parser/generator for C++ with both SAX/DOM style API
```